### PR TITLE
Component system: lots of foundational pieces

### PR DESCRIPTION
**Pull request by marcelkornblum**
Add 
* global styling placeholder, and way to temporarily fake it for parts of pages
* CSS root variables
* Layout primitives for composition
* CTA and Link List components in rigth place
* removed element from API

- [ ] JIRA ticket referenced in title
- [x] Title is clear and concise
- [ ] Description gives any relevant detail
- [x] Tests are up to date
- [x] Documentation is up to date

### CODE REVIEW

File: src/dw_design_system/dwds/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Digital Workspace design system
+
+A basic and simple design system for Digital Workspace content.
+
+## Overall aproach
+
+Folder structure and layout - components and elements
+
+Wehn is it a component
```
Comment (alsyx) [inline src/dw_design_system/dwds/README.md:9]:
```suggestion
When is it a component
```

**Review by alsyx — APPROVED** [review]
Very helpful, thank you.

File: src/dw_design_system/dwds/components/link_list.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        context = parent_context or {}
+        context.update(
+            {
+                "title": value["title"],
+                "list": [
+                    {"url": page.get_url(), "text": page.title}
+                    for page in value["pages"]
+                ],
+            }
+        )
```
Comment (CamLamb) [inline src/dw_design_system/dwds/components/link_list.py:39]: .update can take kwargs, so the following does the same with less lines/code

```suggestion
        context = parent_context or {}
        context.update(
            title=value["title"],
            list=[
                {"url": page.get_url(), "text": page.title}
                for page in value["pages"]
            ],
        )
```

**Review by CamLamb — APPROVED** [review]
Just added one comment which isn't too important

File: src/dw_design_system/dwds/components/link_list.py (hunk 2 above)
Comment (marcelkornblum) [reply src/dw_design_system/dwds/components/link_list.py:39]: oh nice

**Review by nicopicchio — APPROVED** [review]

**Pull request merged**